main.go

```diff
@@ -125,6 +126,11 @@ func wrapMain() error {
 	vargs := GAE{}
 	workspace := ""
 
+	// If there is an env vars file load it
+	if _, err := os.Stat("/run/drone/env"); err == nil {
+		godotenv.Overload("/run/drone/env")
```

Comment:
I wonder if it would make more sense for this to be `Load` instead of `Overload` so it doesn't overwrite any of the pipeline configured settings?

Reply:
Makes perfect sense, committed that change here https://github.com/nytimes/drone-gae/pull/65/commits/79027e2ee2559b4768869fa38a24267651ea2d5a